charts/kubeclarity/values.yaml

```diff
@@ -296,33 +295,20 @@ kubeclarity-sbom-db:
 #######################################################################################
 
 #######################################################################################
-## KubeClarity Postgres Values
-
+## KubeClarity Internal Postgres Values
+## Use kubeclarity-postgresql if you want this chart to deploy a PostgreSQL instance
 kubeclarity-postgresql:
   enabled: true
 
-  ## Specify posgtresql image
-  # image:
-    # registry: docker.io
-    # repository: bitnami/postgresql
-    # tag: 14.4.0-debian-11-r4
-
-  ## initdb parameters
-  # initdb:
-    ##  ConfigMap with scripts to be run at first boot
-    ##  NOTE: This will override initdb.scripts
-    # scriptsConfigMap
-    ##  Secret with scripts to be run at first boot (in case it contains sensitive information)
-    ##  NOTE: This can work along initdbScripts or initdbScriptsConfigMap
-    # scriptsSecret:
-    ## Specify the PostgreSQL username and password to execute the initdb scripts
-    # user:
-    # password:
-
```

Comment:
for the review to be more clear, can you please bring back the fields here and not at the end?

Reply:
@FrimIdan Updated

Also, comment is removed for postgresql image as I find the template clearer like this (no defaults in the image name in the template).
I can put it back as it was if needed.